cyberdrop_dl/managers/config_manager.py

```diff
@@ -171,14 +172,28 @@ def _verify_settings_config(self) -> None:
         self.settings_data["Runtime_Options"]["console_log_level"] = int(
             self.settings_data["Runtime_Options"]["console_log_level"],
         )
+        #convert paths to str
+        settings_data=copy.deepcopy(self.settings_data)
+        settings_data["Files"]["input_file"] = str(self.settings_data["Files"]["input_file"])
+        settings_data["Files"]["download_folder"] = str(self.settings_data["Files"]["download_folder"])
+        settings_data["Logs"]["log_folder"] = str(self.settings_data["Logs"]["log_folder"])
+        settings_data["Logs"]["webhook_url"] = str(self.settings_data["Logs"]["webhook_url"])
+        settings_data["Sorting"]["sort_folder"] = str(self.settings_data["Sorting"]["sort_folder"])
+        settings_data["Sorting"]["scan_folder"] = str(self.settings_data["Sorting"]["scan_folder"])
```

Comment:
Add a check for `None` cause `scan_folder` could be `None` and it will be saved as `"None"` and sorting will not work cause the folder "None" does not exists